userdetails-cognito/src/main/groovy/au/org/ala/userdetails/CognitoUserService.groovy

```diff
@@ -64,6 +71,8 @@ class CognitoUserService implements IUserService<UserRecord, UserPropertyRecord,
     AWSCognitoIdentityProvider cognitoIdp
     String poolId
     JwtProperties jwtProperties
+    AmazonApiGateway apiGatewayIdp
+    GrailsApplication grailsApplication
```

Comment:
I'd prefer not to inject grailsApplication and inject *Properties classes instead